docs/security/security.md

```diff
@@ -136,7 +136,7 @@ Finally, if you run Docker on a server, it is recommended to run
 exclusively Docker in the server, and move all other services within
 containers controlled by Docker. Of course, it is fine to keep your
 favorite admin tools (probably at least an SSH server), as well as
-existing monitoring/supervision processes (e.g., NRPE, collectd, etc).
+existing monitoring/supervision processes, such as NRPE and collectd.
```

Comment:
Thanks @cpuguy83, thanks @mstanleyjones. Here the "e.g." and "etc." are repeat, and I think the words of mstanleyjones are appropriate. When there is ambiguity, we can use another expression.